core/src/main/java/org/apache/metamodel/schema/ImmutableColumn.java

```diff
@@ -38,12 +38,55 @@
     private final Boolean nullable;
     private final String remarks;
     private final Integer columnSize;
+    private final Integer decimalDigits;
     private final String nativeType;
     private final boolean indexed;
     private final boolean primaryKey;
     private final String name;
     private final String quote;
 
+    /**
+     * Constructs a new {@link ImmutableColumn}.
+     *
+     * @param name
+     *            the name of the column
+     * @param type
+     *            the type of the column
+     * @param table
+     *            the table which the constructed column will pertain to
+     * @param columnNumber
+     *            the column number of the column
+     * @param columnSize
+     *            the size of the column
+     * @param nativeType
+     *            the native type of the column
+     * @param nullable
+     *            whether the column's values are nullable
+     * @param remarks
+     *            the remarks of the column
+     * @param indexed
+     *            whether the column is indexed or not
+     * @param quote
+     *            the quote character(s) of the column
+     * @param primaryKey
+     *            whether the column is a primary key or not
+     */
+    public ImmutableColumn(String name, ColumnType type, Table table, int columnNumber, Integer columnSize,
+                           String nativeType, Boolean nullable, String remarks, boolean indexed, String quote, boolean primaryKey) {
+        this.name = name;
```

Comment:
It is common practice to use constructor chaining when using multiple constructors for a java class, so then you could change this to:

```
        this(name, type, table, columnNumber, columnSize, null, nativeType, nullable, remarks, indexed, quote,
                primaryKey);
```

Reply:
OK, this should be ready for another look.

Reply:
This looks good now.